lib/hexpm/web/views/layout_view.ex

```diff
@@ -42,9 +42,17 @@ defmodule Hexpm.Web.LayoutView do
       tag(:meta, property: "og:title", content: Map.get(assigns, :title)),
       tag(:meta, property: "og:type", content: "website"),
       tag(:meta, property: "og:url", content: Map.get(assigns, :canonical_url)),
-      tag(:meta, property: "og:image", content: static_url(Hexpm.Web.Endpoint, "/images/hex.png")),
+      tag(:meta, property: "og:image", content: static_url(Hexpm.Web.Endpoint, "/images/open-graph.png")),
```

Comment:
So yeah, I think we should go either:

1. just Hex logo
2. Hex logo + "Hex" text to the right. Same as https://github.com/hexpm/hexpm/pull/492/files#diff-58974f2430a70881b1191c4d2c16e898, but with no extra background, padding or any other text

@ericmj ?